### ISSUE TITLE
UI Design for Dashboard

### ISSUE THREAD

**Issue by Vanisha27** [UdacityFrontEndScholarship/quiz-app#50]
### Help us with the design of our dashboard.

The dashboard would include few details about the user and the statistics regarding their current progress in the quizzes.

This is a basic mockup of the same.
![image](https://user-images.githubusercontent.com/36723618/38999883-64afc1fa-4410-11e8-8d48-980f1930451d.png)

Here are a few things to be included on this page - 
1. User's profile picture
2. Full Name
3. Username
4. Statistics for _Practice_, _Single Player_ and _Challenge mode_.
5. Progress in quiz
6. Settings option

Apart from the things mentioned above, _you can include other elements_ which could be relevant here.

Feel free to use your creativity! :slightly_smiling_face: 

**Note** - Please try to submit your ideas by **Saturday**, i.e. , April 21.

**Comment by viditkothari** [comment]
May I please get some guide on your branding / persona?

**Comment by jibin2706** [comment]
User Rank maybe

**Comment by ezioda004** [comment]
@jibin2706 Yes, we plan to include a progression system with Rank and stuff. Please submit your design if you make one!

**Comment by kadumuri1994** [comment]
@creatorisback , as of now, you can consider data from the user statistics table of the below relational model (https://www.lucidchart.com/documents/edit/72a025d2-3e07-4d50-aa47-74e3001e4b46/0?shared=true&) 
Perhaps, @ezioda004  might want to add/remove statistics at a later point.

**Comment by creatorisback** [comment]
@kadumuri1994 lucid chart has permission issue my email is (8826047538r@gmail.com) please provide access so I can view document

**Comment by kadumuri1994** [comment]
@creatorisback , can you check now?

**Comment by creatorisback** [comment]
@kadumuri1994 still can't access doc 
![ww](https://user-images.githubusercontent.com/28557067/39086429-bf8bdbc8-45ae-11e8-9447-75faf83c98e7.PNG)

**Comment by Vanisha27** [comment]
Hey @creatorisback 
For now, we have decided to collect the following user data - 

- Full Name
- Profile picture
- Username
- Statistics for 
-- _Practice mode_ : Highscore, Avg score, Avg answer time.
-- _Challenge mode_ : Wins, Draws, Losses, Avg score, Avg answer time
- Number of matches played
- Total Score

Please do note that these can be changed later.
:slightly_smiling_face:

**Comment by creatorisback** [comment]
That's why i want to discuss because bring change in UI becomes harder if the change is major so let's work together on this so we could figure out stats and type of data to be represented....A leaderboard should not just be appealing but usable too..:)

**Comment by Vanisha27** [comment]
You mean dashboard 😅

We have an ER diagram prepared so there shouldn't be really major changes. But yeah we can discuss before proceeding. 🙂

**Comment by Saichethan** [comment]
after signup a user dashboard should be generated for every particular user how are we doing it

**Comment by viditkothari** [comment]
Couple of things that I want to ask regarding this:

1. Have you set the scope for this? Without setting a scope no one can design this appropriately.
2. Do you have responsive application in the current scope? If so, then the design needs to be more than just well thought
3. Do you have some branding style guide / persona set for your app? By persona I mean look n feel?
4. **Specifically in the current scope of development** are you looking forward to include any other information in this dashboard screen apart from the one shown in the Issue-wireframe ?

**Comment by Vanisha27** [comment]
Hey @viditkothari, 
Sorry for my late response.
I'll try to answer your queries.

We have set a scope for this but not a very rigid one as for now. Also, we don't have a fixed criterion for the design, so any relevant additions can be done apart from the ones already mentioned.

Yes, we might include some more information but there shouldn't be any changes on the screen which would be difficult to accomodate.

As for the look n feel, any ideas from the contributors are welcome. I'll attach the new mockup made by Parul for reference.
![dashboard detailed](https://user-images.githubusercontent.com/36723618/39209204-dcc9b126-4822-11e8-8a74-0ddb53fb6f15.png)

**Comment by viditkothari** [comment]
Hi @Vanisha27,
Colors which I think should suit the persona:
![mini Brand color Guide: ](https://user-images.githubusercontent.com/409625/39229673-4d425940-4881-11e8-8219-efc88dab8323.jpg)

Sample use of graphic element: in the UI and page footer.
![quizapp_2](https://user-images.githubusercontent.com/409625/39229660-3f2d9342-4881-11e8-86b1-386037e796b0.jpg)

About the dashboard, I think going with centered-aligned circular profile avatar should be better.
followed by centered status text (like Score, no. of games, etc.)

Followed by linebars instead of circular / pie graphs to accommodate all the scores in one screen. **Scrolling isn't so much loved :grinning: **

Let me see if I can design the UI, but I hope you get the idea.

**Comment by Vanisha27** [comment]
Hey @viditkothari 
Thank you for the suggestions. They are really helpful. :innocent: 
Use of graphic element in UI looks good. :ok_hand: 

As for the alignment of profile avatar, we're going for left-aligned since we want to put only the statistics in the centre.
Also, if pie graphs are used, I think only one would be enough to show many details.

Yepp scrolling should not be needed. In the mockup attached in my previous comment, one screen is for _Challenge mode_ and the next one for _Practice mode_. One screen should be enough for scores and other data.
:slightly_smiling_face:

**Comment by viditkothari** [comment]
Thanks :)
I expressed concern for scrolling because there is significant possibility of quiz app being used from devices which are still running in 1260 or 1366 dimensions of lower and for mobile devices when the circular graphs start to stack themselves, user may be forced to scroll (lowering the size of the graphs will make it less legible. Unless you include a font in your brand guide which looks great even in tiny dimension and you size up the graphs **precisely**.

*Also*, I think you can remove the word *Total* from the score labels and use *Avg.* instead of average.

**Comment by Vanisha27** [comment]
Will surely take these things into consideration. Thanks a lot. :)
